resources/measures/ResidentialHotWaterFixtures/measure.rb

```diff
@@ -404,7 +404,7 @@ def remove_existing(runner, space, obj_names)
       found = false
       obj_names.each do |obj_name|
         next if not space_equipment.name.to_s.start_with? obj_name
-        next if space_equipment.name.to_s.include? "=" # TODO: Skip dummy distribution objects; can remove once we are using AdditionalProperties
+        next if space_equipment.name.to_s.include? "dist" # TODO: Skip dummy distribution objects; can remove once we are using AdditionalProperties
```

Comment:
How about this instead?
`next if space_equipment.additionalProperties.getFeatureAsDouble('dist_hw').is_initialized`

Reply:
Yeah, that looks much better! I didn't feel good about string check, but decided to follow suit anyway, haha!